### PR TITLE
Add throbber on create and reset

### PR DESCRIPTION
**Pull request by muffinresearch**
Show the throbber on pin creation / reset immediately.

### CODE REVIEW

File: public/js/views/create-pin.js [hunk 1]
```diff
@@ -91,6 +91,7 @@ define([
         if (e.target && e.target.nodeName === 'BUTTON') {
           $(e.target).prop('disabled', true);
         }
+        app.throbber.render();
```
Comment (kumar303) [inline public/js/views/create-pin.js]: what if you just called this at the top of the function: https://github.com/muffinresearch/spartacus/blob/loading-on-reset-and-create/public/js/views/create-pin.js#L104 ? That would handle the case where it's called recursively: https://github.com/muffinresearch/spartacus/blob/loading-on-reset-and-create/public/js/views/create-pin.js#L123

**Comment by kumar303** [comment]
r+wc

File: public/js/views/create-pin.js (hunk 1 above)
Comment (muffinresearch) [inline public/js/views/create-pin.js]: Any error will clear the throbber. https://github.com/mozilla/spartacus/blob/master/public/js/views/error.js#L39

Also adding it to `handleSubmit` means it'll be inherited by reset-pin which overrides submitData.

Comment (kumar303) [inline public/js/views/create-pin.js]: I see. Sounds good. r+

Comment (muffinresearch) [inline public/js/views/create-pin.js]: I'll make sure I test it once it lands.

**Pull request merged**